docs/docs/references/configuration/cli/trivy_filesystem.md

```diff
@@ -43,6 +43,7 @@ trivy filesystem [flags] PATH
       --ignore-unfixed                   display only fixed vulnerabilities
       --ignored-licenses strings         specify a list of license to ignore
       --ignorefile string                specify .trivyignore file (default ".trivyignore")
+      --include-dev-deps                 include developer dependencies in the report. (Currently only Npm supports this flag)
```

Comment:
I think "npm" is correct.
https://docs.npmjs.com/about-npm

```suggestion
      --include-dev-deps                 include developer dependencies in the report. (Currently only npm supports this flag)
```

Reply:
Changed in https://github.com/aquasecurity/trivy/pull/4700/commits/f4a1e2002f31d401323e09fb747d58c57b97d4a3